src/problem1.js

```diff
@@ -1,6 +1,38 @@
 function problem1(pobi, crong) {
-  var answer;
-  return answer;
+  if (!isValidPage(pobi) || !isValidPage(crong)) {
+    return -1;
+  }
+  else {
+    pobiScore = Math.max(getMaxScore(pobi[0]), getMaxScore(pobi[1]));
+    crongScore = Math.max(getMaxScore(crong[0]), getMaxScore(crong[1]));
+    return getWinner(pobiScore, crongScore);
+  }
```

Comment:
위의 `if` 문 코드 블락 아래로는

암묵적으로 else 가 적용이 된거나 다름 없으니깐

```suggestion
  pobiScore = Math.max(getMaxScore(pobi[0]), getMaxScore(pobi[1]));
  crongScore = Math.max(getMaxScore(crong[0]), getMaxScore(crong[1]));
  return getWinner(pobiScore, crongScore);
```

이렇게 `else` 를 쓰는 거는 불필요할거같아용 ㅎㅎ

이렇게 `else` 쓰면은 오히려 코드의 "depth" 가 불필요하게 깊어지니깐!

Reply:
@p3bble123